src/components/common/formFields/RadioGroup.tsx

```diff
@@ -32,12 +32,13 @@ export interface RadioGroupProps {
     name: string;
     disabled?: boolean;
     onChange: (event) => void;
+    className?: string;
 }
 
 export class RadioGroup extends Component<RadioGroupProps> {
 
     render() {
-        return <div className="form__radio-group">
+        return <div className={`form__radio-group ${this.props.className}`}>
```

Comment:
I think final className will be `form__radio-group undefined` if className prop is not passed.